apps/news-site/news-site-next/dist/config.json

```diff
@@ -5,6 +5,18 @@
                 "src": "https://benchmark-workloads-ad-css-animations.netlify.app/",
                 "width": "970",
                 "height": "250"
+            },
+            "sections": {
+                "3": {
```

Comment:
kinda weird, I know. I am using it as an "index", so inject ad before the 3rd section, ect...
Just easier to read and I don't have to create an array with empty values for index 0, 1, 2,